examples/pytorch_simple.py

```diff
@@ -0,0 +1,174 @@
+            self.dropouts.append(nn.Dropout(dropout))
+            input_dim = output_dim
+
+        self.layers.append(nn.Linear(input_dim, CLASSES))
+
+        # assinging the layers as class variables (PyTorch requirement).
+        for idx, layer in enumerate(self.layers):
+            setattr(self, 'fc{}'.format(idx), layer)
+
+        # assinging the dropouts as class variables (PyTorch requirement).
+        for idx, dropout in enumerate(self.dropouts):
+            setattr(self, 'drop{}'.format(idx), dropout)
+
+    # Forward pass computation function.
+    def forward(self, data):
+        data = data.view(-1, 28 * 28)
+        for layer, dropout in zip(self.layers, self.dropouts):
+            data = F.relu(layer(data))
+            data = dropout(data)
+        return F.log_softmax(self.layers[-1](data), dim=1)
+
+
+def get_mnist():
+    # Load MNIST dataset.
+    train_loader = torch.utils.data.DataLoader(datasets.MNIST(DIR,
+                                                              train=True,
+                                                              download=True,
+                                                              transform=transforms.ToTensor()),
+                                               batch_size=BATCHSIZE,
+                                               shuffle=True)
+    test_loader = torch.utils.data.DataLoader(datasets.MNIST(DIR,
+                                                             train=False,
+                                                             transform=transforms.ToTensor()),
+                                              batch_size=BATCHSIZE,
+                                              shuffle=True)
+
+    return (train_loader, test_loader)
+
+
+def objective(trial):
+
+    # Generate the model.
+    model = Net(trial).to(DEVICE)
+
+    # Generate the optimizers.
+    optimizer_name = trial.suggest_categorical('optimizer', ['Adam', 'RMSprop', 'SGD'])
+    lr = trial.suggest_uniform('lr', 1e-5, 1e-1)
+    optimizer = getattr(optim, optimizer_name)(model.parameters(), lr=lr)
+
+    # Get the MNIST dataset.
+    train_loader, test_loader = get_mnist()
+
+    # Training of the model.
+    model.train()
+    for epoch in range(EPOCHS):
+        for batch_idx, (data, target) in enumerate(train_loader):
+            # Limiting training data for faster epochs.
+            if batch_idx * BATCHSIZE >= N_TRAIN_EXAMPLES:
+                break
+
+            data, target = data.to(DEVICE), target.to(DEVICE)
+
+            # Zeroing out gradient buffers.
+            optimizer.zero_grad()
+            # Performing a forward pass.
+            output = model(data)
+            # Computing negative Log Likelihood loss.
+            loss = F.nll_loss(output, target)
+            # Performing a backward pass.
+            loss.backward()
+            # Updating the weights.
+            optimizer.step()
+
+            # Printing statistics.
+            if batch_idx % LOG_INTERVAL == 0:
```

Comment:
In my opinion, we can remove logging code (here and [L154](https://github.com/pfnet/optuna/pull/366/files#diff-c3bee537c4bf5a7c32e4356765c78687R154)) from this example.
It will make the code a bit simpler and the output of this example a less noisy (users can focus on the log messages emitted by Optuna).